python/lsst/jointcal/jointcal.py

```diff
@@ -135,7 +156,8 @@ def _build_ccdImage(self, dataRef, associations, jointcalControl):
 
         goodSrc = self.sourceSelector.selectSources(src)
 
-        # TODO: NOTE: Leaving the old catalog and comparison code in while we
+        # ------------------
+        # TODO: Leaving the old catalog and comparison code in while we
```

Comment:
I've now wrapped this code in `if useOldStarSelector:` with the note above it describing when it will be removed.